akka-bench-jmh/src/main/scala/akka/util/LruBoundedCacheBench.scala

```diff
@@ -0,0 +1,74 @@
+package akka.util
+
+import java.util
+import java.util.concurrent.TimeUnit
+
+import akka.remote.artery.{ FastHash, LruBoundedCache }
+import org.openjdk.jmh.annotations.{ Param, _ }
+
+import scala.util.Random
+
+@State(Scope.Benchmark)
+@Measurement(timeUnit = TimeUnit.MICROSECONDS)
+class LruBoundedCacheBench {
```

Comment:
This is a rather silly benchmark as there is really nothing to exactly compare to, plus I am not 100% trusting the JMH output on this. I used it more as a sanity check, maybe I just remove this altogether.